modules/preview-react/table/stories/Table.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import {SymbolDoc, Specifications} from '@workday/canvas-kit-docs';
+import {Table} from '@workday/canvas-kit-preview-react/table';
+// examples
+import {Basic} from './examples/Basic';
+import {BasicWithHeading} from './examples/BasicWithHeading';
+import {FixedColumn} from './examples/FixedColumn.tsx';
+
+<Meta title="Preview/Table" component={Table} />
+
+# Canvas Kit Table
+
+`Table` is a simple styled compound component that renders a
+[table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) element. It is used to
+present information in a two-dimensional table comprised of rows and columns of cells containing
+data.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-preview-react
+```
+
+## Usage
+
+### Basic Example with Heading
+
+In this example, users may not want to use a `caption` so they can import
+[Heading](/docs/components-text-heading--basic) or [Text](/docs/components-text-text--basic)
+instead. This will give the user more flexibility around the customization of the title/heading of
+their table.
+
+<ExampleCodeBlock code={BasicWithHeading} />
+
+### Basic Example with Caption
+
+In this example, we built a simple table that includes a `caption`. A `caption` is not required but
+it is good for
+[accessibility](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#accessibility_concerns)
+purposes.
+
+<ExampleCodeBlock code={Basic} />
+
+### Fixed Column and Column Dividers
+
+In this example, users may want to have columns divided and have a fixed left column for their
+headers so users can scroll horizontally on bigger data sets.
+
+<ExampleCodeBlock code={FixedColumn} />
```

Comment:
I feel like column dividers warrants its own example separate from fixed columns (though it does make sense to use them together).

Reply:
Column dividers are just borders on the cells and headers. I wonder if it does need an example on it's own?